frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunTabParameters.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import {
+  Spinner,
+  EmptyStateVariant,
+  EmptyState,
+  Title,
+  EmptyStateBody,
+} from '@patternfly/react-core';
+import { PipelineRunKF } from '~/concepts/pipelines/kfTypes';
+import {
+  DetailItem,
+  renderDetailItems,
+} from '~/concepts/pipelines/content/pipelinesDetails/pipelineRun/utils';
+type PipelineRunTabParametersProps = {
+  pipelineRunKF?: PipelineRunKF;
+};
+
+const PipelineRunTabParameters: React.FC<PipelineRunTabParametersProps> = ({ pipelineRunKF }) => {
+  if (!pipelineRunKF) {
+    return (
+      <EmptyState variant={EmptyStateVariant.large} data-id="loading-empty-state">
+        <Spinner size="xl" />
+        <Title headingLevel="h4" size="lg">
+          Loading
+        </Title>
+      </EmptyState>
+    );
+  }
```

Comment:
No PipelineRunKF is likely because you're on a page after a run was removed (or you manually navigated to an invalid ID) -- state is pretty much straight passed through (once upon a time) before we had proper error states. It may not be possible anymore 🤔

Reply:
The other tabs used it so i included it. Should it be removed? I have never seen the loading state, but that doesnt mean its not possible lol

Reply:
We definitely don't want to be adding dead code. If the types aren't correct, it's best to attempt to clean up the source of the issue. I couldn't get into a situation that loads this code while testing.

That being said, it can be logged as a separate issue if it's something that prevalent across other tabs.

Reply:
i will play around with it a sec and make an issue if it happens across the board